examples/Migration/composer.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "googleads/google-ads-php-examples-migration",
```

Comment:
Do we really need to include everything here?
Can we just include "require"?

Reply:
The autoload part is needed for classes importing, so it cannot be removed; I'd also keep the require-dev dependency on phpcs and the suggestion for php >=7.1.

I don't have a strong opinion on all the rest, but I feel like including them is not detrimental to the clarity of the project.

Reply:
For autoload, it's probably safe to leave it there, I agree.

For others, it depends on how you would allow people use this example.
We have similar situation back in the SOAP library too.
What we did there is pretty simple, which is to include the SOAP library itself (pretty much like you do here) and that's it. No need to specify the name (as we're not planning to host it in packagist).

Besides, the require-dev, suggestion, license, author (we actually have two author keys here) seems unnecessary to me.
This folder resides in a large google-ads-php repository + it requires the google-ads-php explicitly itself, those pieces of information will be enforced / included / offered to users when they run composer-install.

At least, I would suggest removing the name and description, because we're not hosting this in packagist and it could be confusing in the future.

Reply:
Done.